src/components/LoanCollections/KivaClassicLoanCarouselExp.vue

```diff
@@ -35,15 +42,15 @@
 </template>
 
 <script>
-import KivaClassicBasicLoanCardExp from '@/components/LoanCards/KivaClassicBasicLoanCardExp';
```

Comment:
Need to keep this import as it's still used in the component

Reply:
Nice catch, thanks!